lighthouse-cli/cli-flags.ts

```diff
@@ -12,9 +12,9 @@ const Driver = require('../lighthouse-core/gather/driver.js');
 import {GetValidOutputOptions, OutputMode} from './printer';
 
 export interface Flags {
-  port: number, chromeFlags: string, output: any, outputPath: string, interactive: boolean,
-      saveArtifacts: boolean, saveAssets: boolean, view: boolean, maxWaitForLoad: number,
-      logLevel: string
+  port: number, host: string, chromeFlags: string, output: any, outputPath: string,
+      interactive: boolean, saveArtifacts: boolean, saveAssets: boolean, view: boolean,
+      maxWaitForLoad: number, logLevel: string
```

Comment:
the diff would be a lot clearer (just adding `host: string`) if this had just been appended on the end here, but no biggie :)